docs/_user_guide/best_practices_migrating_from_jupyter.md

```diff
@@ -31,15 +38,13 @@ Before you migrate you have a few option
 * Settle for a Machin learning algorithm over deep learning. These can often times be as effective and use less resources.
 
```

Comment:
a few typos - "option", "kernal" and "Machin" 

this sentence is not grammatically correct - If your usage is going above the CPU limit your program will get shut down, in JupyterHub your kernal will die.